plotly/tools.py

```diff
@@ -1468,9 +1468,9 @@ def _find_intermediate_color(lowcolor, highcolor, intermed):
         diff_1 = float(highcolor[1] - lowcolor[1])
         diff_2 = float(highcolor[2] - lowcolor[2])
 
-        inter_colors = np.array([lowcolor[0] + intermed * diff_0,
```

Comment:
> What is the benefit gained by making it a list and looping through? IME using arrays instead of lists almost always makes things much faster.

I don't necessarily think it's faster or anything like that. I actually just switched it back to this because I wanted it to work with my FF converting code